tools/tsp-client/src/index.ts

```diff
@@ -206,12 +206,14 @@ async function generate({
 async function syncAndGenerate({
```

Comment:
This function might be removed since it doesnt do any additional processing and simply calls sync + generate